testsuite/std/http.c

```diff
@@ -67,33 +66,38 @@ void write_data(int fd) {
 
   tell_object(receiver, sprintf("%O", status));
 
-  socket_write(fd, "GET "+ status[fd]["path"] + " HTTP/1.0\n\r\n\r");
+  socket_write(fd, "GET " + status[fd]["path"] + " HTTP/1.1\nHost: " + status[fd]["host"] + "\n\r\n\r");
```

Comment:
we need to use 1.0 here because we won't support any 1.1 features

Reply:
or HTTP/1.0 but add Host header?